src/reservation.html

```diff
@@ -49,29 +49,29 @@
                     </button>
                 </div>
 
-                <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                <div class="collapse navbar-collapse" id="navbar">
                     <ul class="navbar-nav  header-list">
                         <li class="nav-item header-item">
                             <a class="nav-link header-links  active" aria-current="page" href="landing.html">Home</a>
                         </li>
                         <li class="nav-item header-item">
-                            <a class="nav-link header-links" href="javascript:void(0)">about us</a>
+                            <a class="nav-link header-links" href="landing.html#about">about us</a>
                         </li>
                         <li class="nav-item header-item">
-                            <a class="nav-link header-links" href="javascript:void(0)">gallery</a>
+                            <a class="nav-link header-links" href="landing.html#gallery">gallery</a>
                         </li>
                         <li class="nav-item header-item">
-                            <a class="nav-link header-links" href="javascript:void(0)">Contact us</a>
+                            <a class="nav-link header-links" href="landing.html#contact">Contact us</a>
                         </li>
                         <li class="nav-item header-item">
-                            <a class="nav-link header-links" href="javascript:void(0)">reservation</a>
+                            <a class="nav-link header-links" href="reservation.html">reservation</a>
                         </li>
                         <li class="nav-item header-item d-block d-lg-none">
-                            <a class="nav-link header-links" href="javascript:void(0)">Sign in</a>
+                            <a class="nav-link header-links" href="index.html">Sign in</a>
                         </li>
                     </ul>
                     <div class="d-flex ms-auto">
-                        <button class="btn btn-white btn-signin  me-3 d-none d-lg-block">Sign in</button>
+                       <a href="index.html"> <button class="btn btn-white btn-signin  me-3 d-none d-lg-block" >Sign in</button></a>
```

Comment:
if you are use a tag why addition add button tag inside. please move button class into <A> tag .every where